subsys/settings/src/settings_store.c

```diff
@@ -91,7 +93,7 @@ static void settings_dup_check_cb(char *name, char *val, void *cb_arg)
 /*
  * Append a single value to persisted config. Don't store duplicate value.
  */
-int settings_save_one(const char *name, char *value)
+int settings_save_one(char *name, char *value)
```

Comment:
Does this change mean that it is now invalid to pass C string literals to this function? (such as "myfoo/mybar64")

Reply:
Dosen't mean this - but right, would be better to keep this as 'const char * '.